qutebrowser/browser/network/networkmanager.py

```diff
@@ -241,12 +241,21 @@ def clear_rejected_ssl_errors(self, url):
     @pyqtSlot('QNetworkReply', 'QAuthenticator')
     def on_authentication_required(self, reply, authenticator):
         """Called when a website needs authentication."""
-        answer = self._ask("Username ({}):".format(authenticator.realm()),
+        import netrc
+        net = netrc.netrc()
```

Comment:
It seems this can raise [`NetrcParseError`](https://docs.python.org/3.5/library/netrc.html#netrc.NetrcParseError) - that should probably be handled by showing an error in the statusbar, or using `log.misc.exception(...)`.